src/SelectableBox/tests/SelectableBoxSet.test.jsx

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import { render, screen } from '@testing-library/react';
-import '@testing-library/jest-dom/extend-expect';
+import '@testing-library/jest-dom';
```

Comment:
Before this change I encountered
```
Summary of all failing tests
 FAIL  src/SelectableBox/tests/SelectableBoxSet.test.jsx
  ● Test suite failed to run

    Cannot find module '@testing-library/jest-dom/extend-expect' from 'src/SelectableBox/tests/SelectableBoxSet.test.jsx'

      1 | import React from 'react';
      2 | import { render, screen } from '@testing-library/react';
    > 3 | import '@testing-library/jest-dom/extend-expect';
        | ^
      4 | import userEvent from '@testing-library/user-event';
      5 | import SelectableBox from '..';
      6 |

      at Resolver._throwModNotFoundError (node_modules/jest-resolve/build/resolver.js:491:11)
      at Object.require (src/SelectableBox/tests/SelectableBoxSet.test.jsx:3:1)
```
when running tests locally.

I found [this suggestion](https://github.com/testing-library/react-testing-library/issues/138#issuecomment-1716326742) and it resolved the issue